README.md

```diff
@@ -3,9 +3,10 @@
 The ViewComponent::Storybook gem provides Ruby api for writing stories describing [View Components](https://github.com/github/view_component) and allowing them to be previewed and tested in [Storybook](https://github.com/storybookjs/storybook/) via its [Server](https://github.com/storybookjs/storybook/tree/next/app/server) support.
 
 ## Features
-* A Ruby DSL for writing Stories describing View Components
-* A Rails controller backend for Storybook Server compatible with Storybook Controls Addon parameters
-* Coming Soon: Rake tasks to watch View Components and Stories and trigger Storybook hot reloading
+
+- A Ruby DSL for writing Stories describing View Components
+- A Rails controller backend for Storybook Server compatible with Storybook Controls Addon parameters
+- Coming Soon: Rake tasks to watch View Components and Stories and trigger Storybook hot reloading
```

Comment:
Some changes were made by a markdown linter. `markdownlint` in VSCode. Happy to back these out if undesirable.

Reply:
I like the linting changes